src/alert-manager/config/alert-manager.yaml

```diff
@@ -31,3 +31,6 @@ cluster-utilization:
   configured: False
 use-pylon: False
 repeat-interval: '24h'
+cert-expiration-checker:
+  schedule: '0 0 * * *'
+  alert-residual-days: 30
```

Comment:
This file is loaded as the default config, this part should be configurable in `services-configuration.yaml`, refer to #5281

Reply:
Done.